<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FusionInventory</string>
  </property>
  <property name="windowIcon">
   <iconset resource="FusionInventory.qrc">
    <normaloff>:/fusinv/fusinv.ico</normaloff>:/fusinv/fusinv.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Host to be inventoried</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>Where to Install / Execute FusionInventory Agent</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <widget class="QRadioButton" name="radioButtonLocal">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Local Host (this system)</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButtonRemoteWin">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Remote Windows Host (via RPC Service interface and the $Admin share)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButtonRemoteUnix">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Remote Unix / Linux Host</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBoxRemoteHost">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="title">
            <string>Remote host to be inventoried</string>
           </property>
           <layout class="QFormLayout" name="formLayout_2">
            <item row="1" column="0">
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Host(s)</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="lineEditRemoteHost">
              <property name="toolTip">
               <string>The remote host to be inventoried.</string>
              </property>
              <property name="whatsThis">
               <string>Hostname or IP address of the remote host to be inventoried.</string>
              </property>
              <property name="text">
               <string>hostname.domain.org</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Username</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="lineEditRemoteHostUser">
              <property name="toolTip">
               <string>Username of the remote host to be inventoried.</string>
              </property>
              <property name="whatsThis">
               <string>Username that is used to connect to the remote host to be inventoried.
[Domain/]Username</string>
              </property>
              <property name="text">
               <string>Administrator</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Password</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="lineEditRemoteHostPassword">
              <property name="toolTip">
               <string>Password of the remote host to be inventoried.</string>
              </property>
              <property name="text">
               <string>PlainTextPassword</string>
              </property>
              <property name="echoMode">
               <enum>QLineEdit::PasswordEchoOnEdit</enum>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QCheckBox" name="checkBoxTryNet">
              <property name="text">
               <string>Try the entire ...</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_5">
        <attribute name="title">
         <string>Agent installation folder</string>
        </attribute>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Windows</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEditAgentWin">
           <property name="toolTip">
            <string>Where the FusionInventory-Agent is installed on the target host.</string>
           </property>
           <property name="whatsThis">
            <string>Where the FusionInventory-Agent is installed. Normally C:\Program Files\FusionInventory-Agent\

perl\bin\perl.exe and perl\bin\fusioninventory-agent have to be present in this folder.</string>
           </property>
           <property name="text">
            <string>C:\Program Files\FusionInventory-Agent\</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QToolButton" name="toolButtonAgentWin">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Unix / Linux</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineEditAgentUnix">
           <property name="toolTip">
            <string>Where the FusionInventory-Agent is installed on the target host.</string>
           </property>
           <property name="whatsThis">
            <string>Where the FusionInventory-Agent is installed. Normally, one of the following:
/usr
/usr/local
/usr/fusioninventory-agent
/usr/local/fusioninventory-agent
/opt/fusioninventory-agent


fusioninventory-agent have to be present in this folder. Moreover, if it is a standalone installation, perl/bin/perl have to be present in this folder.</string>
           </property>
           <property name="text">
            <string>/opt/fusioninventory-agent</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QToolButton" name="toolButtonAgentUnix">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>FusionInventory Server</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QFormLayout">
           <property name="fieldGrowthPolicy">
            <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Server</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEditServer">
             <property name="text">
              <string>http://servername/glpi/plugins/fusioninventory/</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Login</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEditServerUser"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Password</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="lineEditServerPassword"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Proxy URL</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="lineEditProxy">
             <property name="text">
              <string>http://domain\username:password@proxyhost:port/</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>Tag</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="lineEditTag">
             <property name="toolTip">
              <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Cantarell';&quot;&gt;Tag can be used to identify an entity in an organizational structure.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="whatsThis">
              <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Cantarell';&quot;&gt;Tag can be used to identify an entity in an organizational structure.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Cantarell';&quot;&gt;For example: &amp;quot;IT-group&amp;quot; or &amp;quot;sale-team&amp;quot;.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="text">
              <string>hostname-department</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>HTTPS (not implemented yet)</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Disable SSL certificat validity</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBoxNoSSLCheck">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>SSL certificat:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="toolButtonSelectCert">
                <property name="text">
                 <string>...</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QLabel" name="labelCaCertFile">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>100</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <italic>true</italic>
                 </font>
                </property>
                <property name="frameShape">
                 <enum>QFrame::NoFrame</enum>
                </property>
                <property name="text">
                 <string>(unknown)</string>
                </property>
                <property name="textFormat">
                 <enum>Qt::AutoText</enum>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonTest">
       <property name="text">
        <string>Perform Inventory</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Save and exit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>pushButtonTest</tabstop>
  <tabstop>pushButtonCancel</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>radioButtonLocal</tabstop>
  <tabstop>radioButtonRemoteWin</tabstop>
  <tabstop>radioButtonRemoteUnix</tabstop>
  <tabstop>lineEditRemoteHost</tabstop>
  <tabstop>lineEditRemoteHostUser</tabstop>
  <tabstop>lineEditRemoteHostPassword</tabstop>
  <tabstop>lineEditServer</tabstop>
  <tabstop>lineEditServerUser</tabstop>
  <tabstop>lineEditServerPassword</tabstop>
  <tabstop>lineEditProxy</tabstop>
  <tabstop>checkBoxNoSSLCheck</tabstop>
  <tabstop>toolButtonSelectCert</tabstop>
 </tabstops>
 <resources>
  <include location="FusionInventory.qrc"/>
 </resources>
 <connections/>
</ui>
